draft-ietf-capport-api.md

```diff
@@ -78,7 +78,7 @@ For example, if the Captive Portal API server is hosted at example.org, the URI'
 
 ## JSON Keys
 
-The Captive Portal API data structures are specified in JavaScript Object Notation (JSON) [RFC7159].
+The Captive Portal API data structures are specified in JavaScript Object Notation (JSON) [RFC7159]. Requests and responses for Captive Portal API SHOULD use the "application/captive+json" media type.
```

Comment:
s/SHOULD// - no need to use the wishy-washy-est 2119 language here

Reply:
the sentence is reading a bit "incomplete" without a shall or should. Also, does it make sense to use a shall/must for the server. i.e a server sending this must set the content type and send this content type.

Reply:
Okay, added a sentence to clarify the client SHOULD and server MUST.